Create 2021-05-14-scopri-come-contribuire-al-miglioramento-dei-serviz… (#994)
italia/developers.italia.it: _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md

---
layout: post
title: Scopri come contribuire al miglioramento dei servizi pubblici digitali del Paese
subtitle: Una nuova iniziativa di Developers Italia e Designers Italia per la Settimana Aperta della PA
date: 2021-05-14 10:00:00 +0100
tags: ogp wizard designers community guida partecipazione
categories: news
author: Riccardo Iaconelli
image: /assets/icons/logo-it.png
locale: it
---

> Chiamare a raccolta tutte le persone che a vario titolo e nelle modalità più diverse si impegnano o si vogliono impegnare a rendere la pubblica amministrazione italiana un luogo più aperto al confronto, più trasparente per i cittadini e più aperto all’innovazione. E’ questo l’obiettivo della **Settimana Aperta della Pubblica Amministrazione** (SAA), iniziativa promossa e coordinata dal Dipartimento della Funzione Pubblica nell’ambito della partecipazione italiana all’Open Gov Week.

Reviewer: ```suggestion
Chiamare a raccolta tutte le persone che a vario titolo e nelle modalità più diverse si impegnano o si vogliono impegnare a rendere la pubblica amministrazione italiana un luogo più aperto al confronto, più trasparente per i cittadini e più aperto all’innovazione. È questo l’obiettivo della **Settimana Aperta della Pubblica Amministrazione** (SAA), iniziativa promossa e coordinata dal Dipartimento della Funzione Pubblica nell’ambito della partecipazione italiana all’Open Gov Week.
```

> L’evento consiste in cinque giornate (dal 17 al 21 maggio 2021) dedicate a sviluppare la cultura e la pratica della trasparenza, della partecipazione e dell’accountability sia nelle amministrazioni pubbliche che nella società.

Reviewer: ```suggestion
L’evento consiste in cinque giornate (dal 17 al 21 maggio 2021) dedicate a sviluppare la cultura e la pratica della trasparenza, della partecipazione e della responsabilità sia nelle amministrazioni pubbliche che nella società.
```

Riconoscendosi a pieno nello spirito che anima questa iniziativa collettiva, Developers Italia e Designers Italia intendono dare il proprio contributo facilitando coloro che desiderano **migliorare i servizi pubblici del Paese**.

> Rilasciamo quindi oggi la prima versione di **uno strumento web **che aiuti gli aspiranti contributori a migliorare, con azioni concrete, il cuore del software libero per la pubblica amministrazione, bene comune di tutto il Paese.

Reviewer: ```suggestion
Rilasciamo quindi oggi la prima versione di **uno strumento web** che aiuti gli aspiranti contributori a migliorare, con azioni concrete, il software libero per la pubblica amministrazione, bene comune di tutto il Paese.
```

Le azioni proposte, in pieno spirito open source, possono essere realizzate già oggi da tutti coloro che desiderano supportare la transizione digitale della Pubblica Amministrazione in ottica di apertura, trasparenza, qualità ed efficienza.

> L’iniziativa è volta anche a risolvere una delle criticità emerse durante il recente [studio di ricerca](https://medium.com/designers-italia/il-valore-della-community-per-designers-italia-e-developers-italia-bd486dff2a6d) realizzato con la community. Gli utenti intervistati hanno infatti dichiarato che non sempre risulta semplice, per coloro che vogliono contribuire, comprendere quali azioni possano avere il maggiore impatto sulla vita dei cittadini, o come impiegare nel modo migliore le proprie competenze. Abbiamo così creato uno strumento facile e veloce, una sorta di __wizard__, che permette di trovare le attività più vicine alle proprie competenze ed inclinazioni.

Reviewer: ```suggestion
L’iniziativa è volta anche a risolvere una delle criticità emerse durante il recente [studio di ricerca](https://medium.com/designers-italia/il-valore-della-community-per-designers-italia-e-developers-italia-bd486dff2a6d) realizzato con la community. Gli utenti intervistati hanno infatti dichiarato che non sempre risulta semplice, per coloro che vogliono contribuire, comprendere quali azioni possano avere il maggiore impatto sulla vita dei cittadini, o come impiegare nel modo migliore le proprie competenze. Abbiamo così creato uno strumento facile e veloce, una sorta di __wizard__, che permette di trovare le attività più vicine alle proprie competenze e inclinazioni.
```
